### PR TITLE
Liquid nomad

### PR DESCRIPTION
**Pull request by mgax**
Improve the docker configuration so that it can pull values directly fom the environment without the need to mount a config file.

### CODE REVIEW

File: snoop/wsgi.py [hunk 1]
```diff
@@ -9,5 +9,6 @@
 
 from django.conf import settings
 from . import defaultsettings
-if not settings.DEBUG and settings.SECRET_KEY == defaultsettings.SECRET_KEY:
-    raise RuntimeError("Please change the default SECRET_KEY setting")
+if settings.SECRET_KEY == defaultsettings._default_secret_key:
```
Comment (mugurrus) [inline snoop/wsgi.py]: Why split in 2 branches?

Comment (mgax) [reply snoop/wsgi.py]: To avoid a long line, so that it's easier to read.

Comment (mugurrus) [reply snoop/wsgi.py]: To me it was perfectly readable as it was. Adding more branches increases complexity, while consolidating them reduces it.

Comment (mgax) [reply snoop/wsgi.py]: Ok, how about now?

Comment (mugurrus) [reply snoop/wsgi.py]: OK

**Comment by mgax** [comment]
@mugurrus Do you think this could break `docker-setup`? If not, I will merge.

**Pull request merged**